Software/bsw/static/Mcal/CAN/src/Can.c

```diff
@@ -914,19 +925,18 @@ uint8 controllerId; /*variable to count controllers number*/
 #if (CanDevErrorDetect == STD_ON)
         if (ControllerState[controllerId] == CAN_CS_UNINIT)
```

Comment:
Can controllerId is not initilaized, we should loop over the configured CAN controllers

Reply:
Sorry, but I didn't get your comment.
What you need here to be changed?